src/app/components/stepper/stepper.component.css

```diff
@@ -0,0 +1,22 @@
+section.vertical-stepper {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+}
+
+section.vertical-stepper form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+section.horizontal-stepper {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+section.horizontal-stepper form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
```

Comment:
Could use the utilityclasses also, but this is totally fine

Reply:
True! This file was mostly copy paste from the react repo. I think I'll leave it, it's fine as you say :)